documentation-test/carlos-doc.shs

```diff
@@ -0,0 +1,1376 @@
+ReadFile
```

Comment:
is this the complete file?
If so delete the jsonl since we can generate it.
I think it's a good approach cos this file is more editable!
Also should .md not .shs